bin/batnode-sample.js

```diff
@@ -64,5 +77,20 @@ if (bat_sample.upload) {
     console.log(chalk.yellow('sample node3 downloads files from sample node1/node2'));
     sendDownloadMessage();
   }
-
-} 
\ No newline at end of file
+  
+} else if (bat_sample.audit) {
+  client = cliNode.connect(1800, 'localhost');
+  
+  console.log(chalk.yellow('You can audit file to make sure file integrity'));
```

Comment:
I was thinking that messages like this could live in a constants folder. So it might look like `constants/messages.js`. We don't necessarily need to do this now, so I created an [issue](https://github.com/WilfredTA/batnode_proto/issues/22) for it.